buildSrc/src/main/kotlin/io.customer/android/Versions.kt

```diff
@@ -1,6 +1,11 @@
 package io.customer.android
 
 object Versions {
+    // Android SDK versions
+    const val COMPILE_SDK = 34
```

Comment:
Is the jump to `34` deliberate?

Reply:
yes, since app running 34, Android 14 are already operational with our SDK, gives a major indication that the SDK is compatible with 34